internal/handlers/handler_fuzz_test.go

```diff
@@ -17,14 +17,55 @@ package handlers
 import (
 	"bufio"
 	"bytes"
+	"context"
 	"testing"
 
 	"github.com/stretchr/testify/require"
+	"go.uber.org/zap"
+	"go.uber.org/zap/zaptest"
 
+	"github.com/FerretDB/FerretDB/internal/handlers/pg"
+	"github.com/FerretDB/FerretDB/internal/handlers/pg/pgdb"
 	"github.com/FerretDB/FerretDB/internal/util/testutil"
 	"github.com/FerretDB/FerretDB/internal/wire"
 )
 
+type setupOpts struct {
+	noLogging bool
+	poolOpts  *testutil.PoolOpts
+}
+
+// setup creates shared objects for testing.
+//
+// Using shared objects helps us spot concurrency bugs.
+// If some test is failing and the log output is confusing, and you are tempted to move setup call to subtest,
+// instead run that single test with `go test -run test/name`.
+func setup(t testing.TB, opts *setupOpts) (context.Context, *pg.Handler, *pgdb.Pool) {
```

Comment:
I don't think that all should be in that file – that one is about fuzzing

Reply:
Should I leave that code in `handler_test.go`?

Reply:
Ah, I see. Yeah, let's leave it there for now